### PR TITLE
ED-2139 update pre-defined list of companies

### PR DESCRIPTION
**Pull request by kowalcj0**
We can observe fairly frequently discrepancies between our pre-defined list of companies and data we're getting from Companies House API. Due to this we have to sync our "fixtures" regularly.
This adds a **make** target `functional_update_companies` which refreshes existing data and replaces inactive companies with new ones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,24 @@
 * Python3
 * [Docker >= 1.10](https://docs.docker.com/engine/installation/)
 * [Docker Compose >= 1.8](https://docs.docker.com/compose/install/)
+* libpq-dev -> `sudo apt install libpq-dev`
```
Comment (konradko) [inline README.md]: this is not gonna work on OSX - what is the requirement? Can it just be included in the docker image?

File: tests/functional/features/pages/utils.py [hunk 2]
```diff
@@ -309,6 +316,65 @@ def load_companies() -> CompaniesList:
...
+    doesnt_exit_counter = 0
+    inactive_counter = 0
+    registered_counter = 0
+    for index, company in enumerate(companies):
+        progress_message = ("Updating company {} out of {}: {} - {}".format(
+            index + 1, companies_number, company.number, company.title))
+        blue(progress_message)
+        exists, active, is_registered = False, False, False
+        ch_result = int_api_ch_search.search(company.number)
+        if len(ch_result) == 1:
```
Comment (konradko) [inline tests/functional/features/pages/utils.py]: what is the result when a company does not exist?

Comment (kowalcj0) [reply tests/functional/features/pages/utils.py]: @konradko the script will find a new active and unregistered company and replace the non-existing one. same thing will happen to `inactive` companies

File: README.md (hunk 1 above)
Comment (kowalcj0) [reply README.md]: @konradko Psycopg it is a wrapper for the libpq, the official PostgreSQL client library: http://initd.org/psycopg/docs/install.html
according to the answers to related question on stackoverflow: https://stackoverflow.com/questions/10132274/install-libpq-dev-on-mac-os-x#10145243
to install libpq on osx:
```bash
#Try installing postgres with brew first
brew install postgresql
#Then install pg 
gem install pg
```

The `functional_update_companies` target should be executed manually, when we decide to refresh the test data. Thus I don't think that we should include it in the docker image.

File: tests/functional/features/pages/utils.py (hunk 2 above)
Comment (kowalcj0) [reply tests/functional/features/pages/utils.py]: PS. this command takes good few minutes to run (usually 3+ minutes) as it needs to call CH API and FAB UI approx. 200 times. So there's no point to run it before each test run but once in a while when we see too many discrepancies between our test data and data returned from CH API

**Pull request merged**